packages/subproviders/src/globals.d.ts

```diff
@@ -3,6 +3,59 @@
 // tslint:disable:async-suffix
 // tslint:disable:completed-docs
 
+// Ethereumjs-tx declarations
+
+// Ledgerco declarations
+interface ECSignatureString {
+    v: string;
+    r: string;
+    s: string;
+}
+interface ECSignature {
+    v: number;
+    r: string;
+    s: string;
+}
+interface ECSignatureBuffer {
+    v: number;
+    r: Buffer;
+    s: Buffer;
+}
+
+interface LedgerTransport {
+    close(): Promise<void>;
+}
+
+declare module '@ledgerhq/hw-app-eth' {
+    class Eth {
+        public transport: LedgerTransport;
+        constructor(transport: LedgerTransport);
+        public getAddress(
+            path: string,
+            boolDisplay?: boolean,
+            boolChaincode?: boolean,
+        ): Promise<{ publicKey: string; address: string; chainCode: string }>;
+        public signTransaction(path: string, rawTxHex: string): Promise<ECSignatureString>;
+        public getAppConfiguration(): Promise<{ arbitraryDataEnabled: number; version: string }>;
+        public signPersonalMessage(path: string, messageHex: string): Promise<ECSignature>;
+    }
+    export default Eth;
+}
+
+declare module '@ledgerhq/hw-transport-u2f' {
+    export default class TransportU2F implements LedgerTransport {
+        public static create(): Promise<LedgerTransport>;
+        public close(): Promise<void>;
+    }
+}
+
+declare module '@ledgerhq/hw-transport-node-hid' {
```

Comment:
Move all of those declarations to DefinitelyTypes

Reply:
These are now removed, have they already been submitted to DT? What's the process?

Reply:
@LogvinovLeon are they complete enough to move to DT?

Reply:
There are no restrictions on completeness. We can move partial definitions there.